ws-client/ws.q

```diff
@@ -21,4 +21,13 @@ open0:{[x;y;v]
 
 open:{neg first open0[x;y;.ws.VERBOSE]}                                 //return neg handle for messaging
 
+.ws.close:{[h]
+  h:abs h;
+  if[all(h in key .ws.w;h in key .z.W);hclose h];                       //close handle if h is found both in .ws.w and .z.W (all opened handles)
+  .ws.w:.ws.w _ h;                                                      //remove h from .ws.w
+  .z.wc h;                                         //remove h from .ws.servers
+ }
+
+.ws.closea:{.ws.close each (0!.ws.w)[`h]}                            //close all opened websockets
```

Comment:
Just need to realign these comments and then I'll merge this :+1:

Reply:
Ok, resolved :)